pypeit/spectrographs/gemini_flamingos.py

```diff
@@ -0,0 +1,340 @@
+        par['calibrations']['darkframe']['exprng'] = [20, None]
+        par['scienceframe']['exprng'] = [20, None]
+
+        # Scienceimage parameters
+        #par['scienceimage']['sig_thresh'] = 5
+        #par['scienceimage']['maxnumber'] = 2
+        par['scienceimage']['skysub']['sky_sigrej'] = 5.0
+        par['scienceimage']['findobj']['find_trim_edge'] = [10,10]
+        # Always flux calibrate, starting with default parameters
+        par['fluxcalib'] = pypeitpar.FluxCalibratePar()
+        # Do not correct for flexure
+        par['flexure'] = None
+
+        return par
+
+    def config_specific_par(self, scifile, inp_par=None):
+        """
+        Modify the PypeIt parameters to hard-wired values used for
+        specific instrument configurations.
+
+        .. todo::
+            Document the changes made!
+
+        Args:
+            scifile (str):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`pypeit.par.parset.ParSet`: The PypeIt paramter set
+            adjusted for configuration specific parameter values.
+        """
+        par = self.default_pypeit_par() if inp_par is None else inp_par
+        # TODO: Should we allow the user to override these?
+
+        if self.get_meta_value(scifile, 'dispname') == 'JH_G5801':
+            par['calibrations']['wavelengths']['method'] = 'full_template'
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'Flamingos2_JH_JH.fits'
+        elif self.get_meta_value(scifile, 'dispname') == 'HK_G5802':
+            par['calibrations']['wavelengths']['method'] = 'full_template'
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'Flamingos2_HK_HK.fits'
+        # Return
+        return par
+
+
+
+    def check_frame_type(self, ftype, fitstbl, exprng=None):
+        """
+        Check for frames of the provided type.
+        """
+        good_exp = framematch.check_frame_exptime(fitstbl['exptime'], exprng)
+        if ftype in ['pinhole', 'bias']:
+            # No pinhole or bias frames
+            return np.zeros(len(fitstbl), dtype=bool)
+        if ftype in ['pixelflat', 'trace']:
+            return good_exp & (fitstbl['idname'] == 'FLAT')
+        if ftype == 'standard':
+            return good_exp & (fitstbl['idname'] == 'OBJECT')
+        if ftype == 'science':
+            return good_exp & (fitstbl['idname'] == 'OBJECT')
+        if ftype in ['arc', 'tilt']:
+            return good_exp & (fitstbl['idname'] == 'OBJECT')
+        msgs.warn('Cannot determine if frames are of type {0}.'.format(ftype))
+        return np.zeros(len(fitstbl), dtype=bool)
+
+    def bpm(self, filename, det, shape=None):
```

Comment:
If you are just using the empty one, remove this method.

Reply:
Removed.